modules/core/context.js

```diff
@@ -431,7 +455,7 @@ export function coreContext() {
 
     // Debounce save, since it's a synchronous localStorage write,
     // and history changes can happen frequently (e.g. when dragging).
-    context.debouncedSave = _debounce(context.save, 350);
+    context.debouncedSave = _debounce(context.save, 100);
```

Comment:
Is there a reason why you changed this?  (If anything I think we could slow down the saves, not speed them up)